app/src/picovr/java/org/mozilla/vrbrowser/PlatformActivity.java

```diff
@@ -146,7 +147,20 @@ protected void onDestroy() {
 
     @Override
     public void onBackPressed() {
-        // Eat the back button.
+        super.onBackPressed();
```

Comment:
I don't think we should do this until we have UI to prevent accidental exit. The home button always works.

Reply:
Reverted

Reply:
I guess we want a v10 task to add this UI, right?

Reply:
I'm not sure if we can get it into v10? Depends on how quick we can get a design. Although I would be okay with creating a place holder. I guess we could always just exit which is what android apps do but I've always hated that about android.